jenkins/ci.suse.de/cloud-gating.yaml

```diff
@@ -90,15 +91,23 @@
           cloudsource: stagingcloud9
           ses_enabled: true
           update_after_deploy: false
+          tempest_filter_list: "\
+            keystone,swift,glance,cinder,neutron,nova,barbican,fwaas,lbaas,\
+            heat,magnum,manila"
+          # Delete when adding to the cloud 9 gating job
+          cloud_env: cloud-gate9-slot
```

Comment:
Let's use another label for those 3 slots dedicated only for these jobs, to leave the cloud9 gating slots untouched.

Reply:
done!